docs/release-notes.rst

```diff
@@ -74,11 +74,8 @@ Upcoming Version (Not Yet Released)
    in the "stable" docs (corresponding to the last release) but will be visible in the
    "latest" docs (corresponding to the master branch).
 
-Bug Fixes
-.........
-
-- Fixed a bug where Bionic could crash when using parallel execution after manually
-  deleting some persisted artifacts (such as with the :ref:`Cache API <cache-api>`).
+0.10.0 (Feb 12, 2021)
```

Comment:
Let's comment out the `Upcoming Version` heading above so it doesn't appear in stable docs.